src/bermudan.cpp

```diff
@@ -162,8 +165,8 @@ Rcpp::List bermudanSwaptionEngine(Rcpp::List rparam,
     QuantLib::Rate dummyFixedRate = 0.03;
     boost::shared_ptr<QuantLib::IborIndex> indexSixMonths(new QuantLib::Euribor6M(rhTermStructure));
         
-    QuantLib::Date startDate = calendar.advance(settlementDate, 1, QuantLib::Years, floatingLegConvention);
-    QuantLib::Date maturity = calendar.advance(startDate, 5, QuantLib::Years, floatingLegConvention);
+    //QuantLib::Date startDate = calendar.advance(settlementDate, 1, QuantLib::Years, floatingLegConvention);  //took out hard coded 
+    //QuantLib::Date maturity = calendar.advance(startDate, 5, QuantLib::Years, floatingLegConvention);         //dates
```

Comment:
It's hard to make something hard coded backward compatible. I could go in and check if there's a null argument and replace it with 1/5  for startDate/maturity, but how many are using this in order to price a very specific expiration/maturity pair?

On the other hand, I modified the documented example to line up with the original 1/5 tenors tenor results that were the default results (though nowhere in the documentation are they listed as the assumed values).